COLLABORATOR_GUIDE.md

```diff
@@ -127,7 +127,7 @@ can be fast-tracked and may be landed after a shorter delay:
 * Focused changes that affect only documentation and/or the test suite.
   `code-and-learn` and `good-first-issue` pull requests typically fall
   into this category.
-* Changes that revert commit(s) and/or fix regressions.
+* Changes that fix regressions.
```

Comment:
It would be better if we include the cases listed by @targos and in https://github.com/nodejs/node/pull/17332#issuecomment-347463150 here.